### DISCUSSION TITLE
wdio config on version 7 failing because it wants me to use the chromedriver service

### DISCUSSION THREAD

**Discussion by ItsASine** [webdriverio/webdriverio#6477] [Q&A] [answered]
It looks like `wdio config` is still recommending the `wdio-chromedriver-service` in v7 when it has a dependency on v6. I get that it's through a third party, but since it was the configuration `wdio` recommended, now my tests don't run on all systems if I install all the other packages manually and use my v6 config rather than going through the wizard.

What else do I need to do to get wdio working in v7 with Chrome since I can't use the chromedriver service? I tried installing `@wdio/selenium-standalone-service` and just swapping the service value with `['selenium-standalone', { drivers: { chrome:  'latest' } }]`, but that didn't actually spin up a selenium server for tests to run

<details>
<summary>Edit: I did get my tests working without the service</summary> but that's not a terribly useful option when the wdio config still wanted me to use the service. It ends up I did some modern js tweaks while fixing my v6 tests to work with v7. Older versions of node didn't like that. Somehow the tests work without the service and my v6 config, and I'm not questioning it haha
</details>

**Comment by christian-bromann** [comment, marked as answer]
@ItsASine just created an issue in the mentioned project: https://github.com/atti187/wdio-chromedriver-service/issues/44. Would you like to take a stab at it? Maybe we can get it updated faster then.

**Reply by ItsASine** [reply]
Looks like it's already addressed! Thank you 👍🏻
